phpcs.xml.dist

```diff
@@ -21,10 +23,27 @@
     <!-- PHP Rules -->
     <rule ref="PHPCompatibility" />
     <config name="testVersion" value="5.2-"/>
+    <rule ref="Squiz.PHP.DisallowMultipleAssignments.Found">
+        <severity>0</severity>
+    </rule>
 
     <!-- WordPress Rules -->
     <rule ref="WordPress">
         <exclude name="WordPress.VIP"/>
+        <!-- In the API, we process the requests. Nonce verification is to be done
+            at the form processing level prior to calling our API. -->
+        <exclude name="WordPress.CSRF.NonceVerification.NoNonceVerification"/>
```

Comment:
I would prefer adding the exclude at the file line level using whitelisting comment, for example:
``` PHP
return $_POST; // phpcs:disable WordPress.CSRF.NonceVerification.NoNonceVerification for X reason.
```
or it my be a bit less specific
``` PHP
return $_POST; // @codingStandardsIgnoreLine for X reason.
```

Reply:
But that's not available yet until PHPCS 2.3 or something is released.  Once that's released, then we can use comment annotations.

Reply:
As far as I am aware, `// @codingStandardsIgnoreFile`, `// @codingStandardsIgnoreStart`, `// @codingStandardsIgnoreEnd` and `// @codingStandardsIgnoreLine` are so we can use that until PHPCS 3.2.0 is released.